test/utilities/notifications.ts

```diff
@@ -7,6 +7,7 @@
 
 import { Workbench } from 'wdio-vscode-service';
 import { pause } from './miscellaneous';
+import * as utilities from '../utilities';
```

Comment:
```suggestion
```

Reply:
Retained the import utilities in order to print the error message in the try/catch block of dismissAllNotifications().  The error occasionally occurred after adding await to utilities.dismissAllNotifications so I added a try/catch block to note the error and make the flow proceed.

Reply:
You only need to import log from miscellaneous! :)